snippets/java/LongestCommonSubsequence.md

```diff
@@ -0,0 +1,91 @@
+# Longest Common Subsequence
```

Comment:
Edit the title of this pull request to Longest Common Subsequence

Reply:
And update the PR message accordingly too